scss/5-objects/_prose.scss

```diff
@@ -57,6 +57,20 @@
   a[href^="https://"]:not([href*="citizensadvice.org.uk"])::after,
   a[href^="http://"]:not([href*="citizensadvice.org.uk"])::after
   {
-    @extend %cads-hyperlink--external-icon;
+    @media not print {
+      @include cads-hyperlink--external-icon;
+    }
+  }
+}
+
+@media print {
+  a[href]::after {
+    content: (' (' attr(href) ')');
+    display: inline;
+  }
+
+  a[href^='/']::after {
+    content: (' (https://www.citizensadvice.org.uk' attr(href) ')');
```

Comment:
Not sure I follow this line? Will this prepend `https://www.citizensadvice.org.uk` to any links starting with `/` i.e. internal?

Reply:
That's right - currently we just render relative links, so when they are printed they aren't much use.  This just adds that to the start of any internal links when a page is printed.  Shall I pop a comment in?

Reply:
Might be worth a quick comment to that effect, ta 👍